templates/login.ubuntu14.erb

```diff
@@ -0,0 +1,17 @@
+auth     optional   pam_faildelay.so  delay=3000000
```

Comment:
This data should be in the manifest and the template should just iterate over the array from the manifest as the other templates do